### ISSUE TITLE
TensorFlow 1.13.1 GPU (CUDA 10.0, cuDNN 7.5, TensorRT 5; MKL), Python 3.7, Buntu 18.04

### ISSUE THREAD

**Issue by evdcush** [yaroslavvb/tensorflow-community-wheels#101]
|  TF  |   Py  | CUDA | cuDNN | TensorRT |        AditionalOpts        | Opt Flags                               |
|:----:|:-----:|:----:|:-----:|:--------:|--------------------|-----------------------------------------|
| 1.13.1 | 3.7.2 | 10.0 |  7.5  |    5.0   | :heavy_check_mark: XLA JIT<br/>:heavy_check_mark: MKL | * SSE4.1, 4.2<br/>* AVX, AVX2<br/>* FMA |

**Release link**: https://github.com/evdcush/TensorFlow-wheels/releases/tag/tf-1.13.1-py37-gpu-10.0

**Wheel link**: https://github.com/evdcush/TensorFlow-wheels/releases/download/tf-1.13.1-py37-gpu-10.0/tensorflow-1.13.1-cp37-cp37m-linux_x86_64.whl

**Install via:**
`pip install --no-cache-dir https://github.com/evdcush/TensorFlow-wheels/releases/download/tf-1.13.1-py37-gpu-10.0/tensorflow-1.13.1-cp37-cp37m-linux_x86_64.whl`

* * *

**Requires:**
- All versions and libraries featured
  - CUDA 10, cuDNN 7.5, TRT 5, Buntu 18.04 OR x64 linux with glibc 2.27 (UNTESTED)
- Nvidia GPU (this wheel optimized for CUDA compute capability 6.1)
- Broadwell or later Intel CPU, (this wheel optimized for `-march=skylake`

**Build info**
* **Compiled on:** `4.15.0-45`, `Xubuntu 18.04`, `GCC-7` with `-march=skylake`
* **Extras:** `XLA JIT` (now supported by default in 1.12), `mkl`, `nccl 2.4`

**Comment by klyuhang9** [comment]
tensorflow-1.13.1-cp37-cp37m-linux_x86_64.whl is not a supported wheel on this platform.?

**Comment by evdcush** [comment]
@klyuhang9

You received this error message when you tried `pip install`?

What is your system setup? Are you on Ubuntu 18.04, with CUDA set up?

**Comment by klyuhang9** [comment]
@evdcush yes,I have done that. Last night I realize the problem may be due to my ubuntu’s original python version is 2.7,even if I have installed python3.7, so I adjust the default to python3.7. 
From other puzzled thought, I hopely want to ask the difference betwwen “pip install tensorflow-gpu==1.13.1”and this wheel installation?

**Comment by evdcush** [comment]
@klyuhang9 

Okay, thank you for following up with the issue; I'm glad to hear it worked for you.

WRT to your other question:

> From other puzzled thought, I hopely want to ask the difference betwwen “pip install tensorflow-gpu==1.13.1”and this wheel installation?


My wheel provides support for [ISA](https://en.wikipedia.org/wiki/Instruction_set_architecture) extensions `AVX2` and `FMA`. So if you have a Broadwell generation or newer Intel CPU, the default tensorflow wheel will not be fully optimized for your CPU's capabilities.

My wheel also uses more recent minor releases for some CUDA libraries, but otherwise **very little** is different.

**Comment by denissato** [comment]
why you used `--no-cache-dir`?

**Comment by farawaytree** [comment]
Could you build **without**
* SSE4.1, 4.2
* AVX, AVX2
* FMA

And **with**

CC 3.0

I could then run on my older Xeon Workstation (no SSE4/AVX etc) with an older GPU!

Thanks!

**Comment by AeroAndZero** [comment]
> tensorflow-1.13.1-cp37-cp37m-linux_x86_64.whl is not a supported wheel on this platform.?

If you are running that command on windows then you have to change the link. This is the link for windows : tensorflow-1.13.1-cp37-cp37m-win_amd64.whl
